meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/u-boot/u-boot-xlnx_2022.01.bb

```diff
@@ -3,7 +3,7 @@ UBOOT_VERSION = "v2022.01"
 UBRANCH = "xilinx-v2022.01-rebase"
 UBOOTURI = "git://github.com/foundriesio/u-boot.git;protocol=https"
 
-SRCREV = "aba57774e4620a03d34f214ab950231880bf3787"
+SRCREV = "b9f0b2dc5baeffbed0f6097e211b709a24011694"
```

Comment:
Rev seems invalid, shouldn't it be https://github.com/foundriesio/u-boot/commit/7c9f39b98b23f77e2cb38b314b5ff5c954f98c04?